bot/cogs/bot.py

```diff
@@ -60,6 +65,36 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.channel.id in self.code_block_channels:
+            if self.allowed[msg.channel.id]-time.time() > 300:
```

Comment:
NameError? come on dude, at least lint out the syntax errors before you commit.

Reply:
http://prntscr.com/ijpqmb my plugin didnt see it :(